packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -1,165 +1,101 @@
-        );
-      });
-      mutate("activeRounds", filteredRounds, false);
-      if (searchQuery) setSearchQuery("");
-    }
-  };
-
-  useEffect(() => {
-    if (searchQuery) {
-      const timeOutId = setTimeout(
-        () => filterProjectsByTitle(activeRounds ?? [], searchQuery),
-        300
-      );
-      return () => clearTimeout(timeOutId);
-    } else {
-      filterGrantRoundByType();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [searchQuery]);
+  const activeRounds = useActiveRounds();
+  const roundsTakingApplications = useRoundsTakingApplications();
+  // const roundsEndingSoon = useRoundsEndingSoon();
 
   return (
-    <>
-      <Navbar showWalletInteraction={true} />
-      <div className=" mx-auto pt-8">
-        <main>
-          <Suspense
-            fallback={
-              <div
-                style={{
-                  width: "100%",
-                  height: "582px",
-                }}
-              />
-            }
-          >
-            <LandingBannerLogo className="w-full h-auto object-cover rounded-t" />
-          </Suspense>
-          <div className="container px-4 md:px-0 md:mx-auto">
-            <h1 className="text-3xl mt-11 mb-10 border-b-2 pb-4">
-              Browse through active rounds
-            </h1>
-            <ApplyNowSection
-              isLoading={applyRoundsLoading}
-              roundOverview={roundsInApplicationPhase}
-            />
-            <ActiveRoundsSection
-              isLoading={activeRoundsLoading}
-              setSearchQuery={setSearchQuery}
-              setRoundType={setType}
-              roundOverview={filterProjectsByTitle(
-                activeRounds ?? [],
-                searchQuery
-              )}
-              searchQuery={searchQuery}
-            />
+    <DefaultLayout showWalletInteraction>
+      <LandingHero />
+      {/* <LandingSection title="Community curation">
+        <div className="grid md:grid-cols-3 gap-x-6">
+          {mock.collections?.slice(0, 4).map((collection, i) => (
+            <div
+              key={collection.id}
+              className={`${i % 3 === 0 ? "md:col-span-2" : ""}`}
+            >
+              <CollectionCard collection={collection} />
+            </div>
+          ))}
+        </div>
+      </LandingSection> */}
+      <LandingSection
+        title="Donate now"
+        action={<ViewAllLink to="#">View all</ViewAllLink>}
+      >
+        <div className="grid md:grid-cols-3 gap-x-6">
+          {activeRounds.data?.slice(0, 4).map((round, i) => (
+            <div
+              key={round.id}
+              className={`${i % 3 === 0 ? "md:col-span-2" : ""}`}
+            >
+              <RoundCard round={round} />
+            </div>
+          ))}
+        </div>
+      </LandingSection>
+      <LandingSection
+        title="Apply for funding"
+        action={<ViewAllLink to="/rounds?status=apply">View all</ViewAllLink>}
```

Comment:
Pass the filters we want to use in the ExploreRoundsPage (status, network, sort by and sort order)